salesagility/SuiteCRM
# Feature codeception module person

Task: SCRM-661

This change includes #4052  #4044 and #4049  - Please review and merge this in first.

## Description



This change adds acceptance tests for the person sugar object. 

## Motivation and Context

Tests person module:
- create person module in module builder
- view person module
- create record with person module
- filter and select record
- edit record
- duplicate record
- delete record
- create person from vcard

## How To Test This

* See [tests/README.md](https://github.com/daniel-samson/SuiteCRM/blob/feature_codeception_basic_module/tests/README.md) and configure your environment appropriately
* Run `composer install`
* Run `codecept run install`
* Run `codecept run tests/acceptance/Core/PersonModuleCest.php`

## Types of changes


### Final checklist
